androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/StopDetailsUnfilteredView.kt

```diff
@@ -40,7 +41,9 @@ fun StopDetailsUnfilteredView(
 
     val analytics: Analytics = koinInject()
 
-    val departures by viewModel.stopDepartures.collectAsState()
+    val groupByDirection by viewModel.groupByDirection.collectAsState()
+    val departures = viewModel.stopDepartures.collectAsState().value
+    val routeCardData = viewModel.routeCardData.collectAsState().value
```

Comment:
I'm confident that `= state.value` is equivalent to `by state` from a functionality perspective and also simplifies the null handling later on (since the getter is allowed to do all sorts of wacky stuff, and under the hood, the `SnapshotMutableStateImpl.value` getter does, but it [WOULD PROBABLY](https://www.rfc-editor.org/rfc/rfc6919#section-5) be idempotent within a single invocation of the component).